includes/integration.drush.inc

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @file
+ * Contains Drush-related hook implementations.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function integration_drush_command() {
+  $items = array();
+  $items['integration-export-document'] = array(
```

Comment:
I would just call it `integration-export`, the `document` object is an internal concept that the final user can easily ignore.